generic/Tensor.c

```diff
@@ -2,7 +2,7 @@
 #define TH_GENERIC_FILE "generic/Tensor.c"
 #else
 
-static void torch_Tensor_(c_readTensorStorageSizeStride)(lua_State *L, int index, int allowNone, int allowTensor, int allowStorage, int allowStride,
```

Comment:
any reason why you removed the allowTensor and allowStride options...? is it because they aren't used anywhere in the library anymore?